.github/workflows/event-release.yml

```
@@ -221,24 +240,21 @@ jobs:
               raise Exception("::error title=No candidates found!")
 
           with ThreadPoolExecutor() as executor:
-              sha_list = executor.map(extract_sha, files)
+              sha_list = list(executor.map(extract_sha, files))
```

Does that actaully work in parallel?
Shouldn't we create the list after we exit the executor scope?
```suggestion
              sha_list = executor.map(extract_sha, files)
          sha_list = list(sha_list)
```